src/DefaultBuilder/test/Microsoft.AspNetCore.TrimmingTests/SlimBuilderDoesNotDependOnX509Test.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Builder;
```

Comment:
Should add copyright to all code files.

Reply:
Oops, lost that when I started building in a scratch project on the side.  I got lint warnings about not using ConfigureAwait, but not about the missing header?

Reply:
> lint warnings

lol